pkg/controller/provider/validation.go

```diff
@@ -267,6 +267,7 @@ func (r *Reconciler) validateSecret(provider *api.Provider) (secret *core.Secret
 				Category: Critical,
 				Message:  err.Error(),
 			})
+			err = nil
```

Comment:
@mansam the req is in the end of the controller but it did not got to that phase as the validation was returning error and critical condition. https://github.com/kubev2v/forklift/blob/b170050734902a37b5e3daa56d433d7d23012c5f/pkg/controller/provider/controller.go#L297-L304